tests/zfs-tests/cmd/stride_dd.c

```diff
@@ -156,7 +156,7 @@ main(int argc, char *argv[])
 	}
 
 	if (seek > 0) {
-		if (lseek(ofd, seek * bsize, SEEK_CUR) == -1) {
+		if (lseek(ofd, (off_t)seek * bsize, SEEK_CUR) == -1) {
```

Comment:
This works, but it would have been cleaner to change the declaration of seek to off_t.

Reply:
Changed to off_t

Reply:
Yeah that's a better idea
I have change both seek and stride to off_t